tests/swagger_generation/test_swagger_generator.py

```diff
@@ -189,22 +189,33 @@ def get_foo(foo_uid):
         def update_foo(foo_uid):
             pass
 
+        # Test using Schema(many=True) without using a nested Field.
+        # https://github.com/plangrid/flask-rebar/issues/41
         @registry.handles(
-            rule='/foos',
+            rule='/foo_list',
             method='GET',
-            marshal_schema={200: ListOfFooSchema()},
-            query_string_schema=FooListSchema(),
+            marshal_schema={200: FooSchema(many=True)},
             authenticator=None  # Override the default!
         )
         def list_foos():
             pass
 
+        @registry.handles(
+            rule='/foos',
+            method='GET',
+            marshal_schema={200: NestedFoosSchema()},
+            query_string_schema=NameAndOtherSchema(),
+            authenticator=None  # Override the default!
+        )
+        def nested_foos():
+            pass
+
         registry.set_default_authenticator(default_authenticator)
 
         host = 'swag.com'
         schemes = ['http']
         consumes = ['application/json']
-        produces = ['application/vnd.plangrid+json']
+        produces = ['application/json']
```

Comment:
(Looks like this should have gone in #19 - figured I'd fix it while I was in here)